addons/pos_sale_report/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Sales Report',
+    'category': 'Sales',
+    'summary': 'Analysis for all sales',
+    'description': """
+Allows to analyse the total sales through report
+================================================
+
+This module allows you to view all your sales report
+from Sales, Point of sale and eCommerce module.
+""",
+    'depends': ['sale', 'point_of_sale', 'website_sale'],
```

Comment:
May be you can remove `sale` from dependencies. Its already depends under `point_of_sale` and `website_sale`. #just